src/components/__tests__/happening-meta-info.test.tsx

```diff
@@ -166,15 +166,21 @@ describe('HappeningMetaInfo', () => {
         const props = happeningMetaInfoProps[0];
         const { getByText } = render(<HappeningMetaInfo {...props} />);
 
-        expect(getByText(new RegExp(`${props.spotRangeCounts?.[0].spots} plasser`))).toBeInTheDocument();
+        expect(getByText(new RegExp(`${props.spotRangeCounts?.[0].spots ?? 'feil'} plasser`))).toBeInTheDocument();
```

Comment:
https://stackoverflow.com/questions/61480993/when-should-i-use-nullish-coalescing-vs-logical-or